src/_locales/ru/messages.json

```diff
@@ -114,7 +114,7 @@
 		"message": "Очистить"
 	},
 	"dictionary_filter_from": {
-		"message": "Оригинальный язык"
+		"message": "Исходный язык"
```

Comment:
I don't agree with this word, let's remove it

Reply:
I'm sorry, i've missed context. I thought it's for translate page, but for dictionary it's exact word!